### PR TITLE
Release/v0.2.0

### PR DESCRIPTION
**Pull request by SvenVw**
## Added
* Adds unit tests
* Adds GitHub Actions for R CMD CHECK and coverage
* Prepare code for package

### CODE REVIEW

File: R/bbwp_field_properties.R [hunk 1]
```diff
@@ -27,16 +27,19 @@
 #' @import OBIC
 #'
 #' @export
-# calculate the relative impact of field properties given its position in the landscape
 bbwp_field_properties <- function(B_SOILTYPE_AGR, B_LU_BRP, B_GWL_CLASS, B_SC_WENR, B_HELP_WENR,
                                   A_CLAY_MI, A_SAND_MI, A_SILT_MI, A_SOM_LOI, A_N_RT,
-                                  A_FE_OX, A_AL_OX, A_P_CC, A_P_AL ,A_P_WA, A_P_SG,
-                                  D_WP, D_RO_R, LSW){
+                                  A_FE_OX, A_AL_OX, A_P_CC, A_P_AL, A_P_WA, A_P_SG,
```
Comment (gerardhros) [inline R/bbwp_field_properties.R:32]: please check if A_P_WA is still needed if you use the latest OBIC. I think it might still be needed

**Review by gerardhros — APPROVED** [review]
update testes and approved.
please check / update the applicability given sector. that needs to be crop specific rather than sector dependent.
And should the impact of measures not be included in the farm score function???

File: R/bbwp_field_properties.R (hunk 1 above)
Comment (SvenVw) [reply R/bbwp_field_properties.R:32]: OBIC v.1.0.4 requires it, so it should be included

**Comment by SvenVw** [comment]
> 
> 
> update testes and approved. please check / update the applicability given sector. that needs to be crop specific rather than sector dependent. And should the impact of measures not be included in the farm score function???

This release is only conversion of the BBWP functions into a R package. New features and enhancements can be included in future releases

**Pull request merged**